commands/create_cluster.go

```diff
@@ -422,6 +432,7 @@ func createDefinitionFromFlags(args addClusterArguments) clusterDefinition {
 // creates a models.V4AddClusterRequest from clusterDefinition
 func createAddClusterBody(d clusterDefinition) *models.V4AddClusterRequest {
 	a := &models.V4AddClusterRequest{}
+	a.AvailabilityZones = float64(d.AvailabilityZones)
```

Comment:
This is currently wrong in the generated client IMO. I made a suggestion in the spec so this gets generated to `int` instead of `float`.

Reply:
Done.